juspay/hyperswitch
# feat(pm_list): Add support for bank transfers in pm list

## Type of Change




## Description

This PR adds support for bank_transfer in payment methods list. The response will be of the form

```
        {
            "payment_method": "bank_transfer",
            "payment_method_types": [
                {
                    "payment_method_type": "ach",
                    "payment_experience": null,
                    "card_networks": null,
                    "bank_names": null,
                    "bank_transfers": {
                        "eligible_connectors": [
                            "stripe"
                        ]
                    }
                }
            ]
        }
```







## How did you test it?

* Create a payment connector with ach enabled.
* Create a payment with confirm set to false
* Call the pm list.


## Checklist
